src/views/CachePurges/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { hot } from 'react-hot-loader';
+import { Component } from 'react';
+import { Switch } from 'react-router-dom';
+import RouteWithProps from '../../components/RouteWithProps';
+import loadable from '../../utils/loadable';
+
+const ViewCachePurges = loadable(() =>
+  import(/* webpackChunkName: 'CachePurges.ViewCachePurges' */ './ViewCachePurges')
```

Comment:
Do you view plural or singular?

Reply:
@eliperelman Not sure I understand what you mean here. Would you say there is something to change here? ViewCachePurges contains a list of active cache purges.